experimental/dds/tree2/src/feature-libraries/defaultFieldKinds.ts

```diff
@@ -599,6 +599,7 @@ const optionalChangeRebaser: FieldChangeRebaser<OptionalChangeset> = isolatedFie
 						childChange: rebaseChild(
 							change.childChange,
 							over.deletedBy === undefined ? undefined : over.childChange,
+							true,
```

Comment:
In the case where `over.fieldChange.wasEmpty` is `true`, it seems a little strange the pass `true` here since no actual deletion occurs. I don't think it'll cause any bugs though.

Reply:
When new content is being put in the field, `wasEmpty` is needed so we can tell whether the field was empty or whether some content is being deleted/overritten.
In the case where content is overritten, we'll have to generate a delta that deletes that content (as well as insert the new content).

Now that I think more about this codepath, I think it's impossible for `wasEmpty` to be `true` here, because `change.childChange !== undefined` and `change.deletedBy === undefined`. Marking this thread as resolved. :)